articles/service-fabric/service-fabric-work-with-reliable-collections.md

```diff
@@ -216,7 +216,27 @@ Furthermore, service code is upgraded one upgrade domain at a time. So, during a
 > While you can modify the schema of a key, you must ensure that your key's hash code and equals algorithms are stable. If you change how either of these algorithms operate, you will not be able to look up the key within the reliable dictionary ever again.

```

Comment:
"you will not be able to look up the key within the reliable dictionary ever again"

Can we change this to note that you would have undefined behavior including corruption of the RC, crashing, and inconsistencies in the data